src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -1506,7 +1506,7 @@ public async IAsyncEnumerable<SourceFile> Load(SessionId id, string[] loaded_fil
             }
         }
 
-        public IEnumerable<SourceFile> AllSources() => assemblies.SelectMany(a => a.Sources);
+        public IEnumerable<SourceFile> AllSources() => assemblies.SelectMany(a => a.Sources).OrderBy(source => source.Url);
```

Comment:
We should definitely cache the source list somewhere, and update it when `assemblies` collection changes. We can sort it at that point too.
This gets used in many places, repeatedly, directly and via `GetFileById`.